docs/source/options/udp_file_options.rst

```diff
@@ -8,11 +8,16 @@ options:
 
     Allow or disallow reusing UDP sockets.
 
-:interface=<addr>, source=<addr>:
+:interface=<addr>:
 
-    Multicast group interface address. Only the packets sent to this address is
+    Multicast group interface address. Only the packets sent to this address are
     received. Default to "0.0.0.0" if not specified.
 
+:source=<addr>:
+
+    Multicast source ip address.  Only the packets sent from this source address
```

Comment:
Remove an extra space between sentences. We usually have only one space between sentences. similar below.